langlib/lang.typedesc/src/main/ballerina/typedesc_lib.bal

```diff
@@ -40,6 +40,17 @@ public type TypeId readonly & record {|
 |};
 
 # Returns the type-ids induced by a typedesc value.
+# 
+# ```ballerina
+# type Error distinct error;
+# 
+# type SampleError distinct (Error & error<record {string msg;}>);
+# 
```

Comment:
```suggestion
# 
# Error.typeIds() ⇒ [{"moduleId":{"organization":"$anon","name":".","platformParts":["0"]},"localId":"Error"}]
#
```

Reply:
Updated in https://github.com/ballerina-platform/ballerina-lang/pull/38983/commits/9f0cc0ee0d5845c8997e16410ce2fc154c6bf887